Source/SwiftLintBuiltInRules/Rules/Idiomatic/PreferKeyPathRule.swift

```diff
@@ -6,11 +6,21 @@ struct PreferKeyPathRule: Rule {
     var configuration = PreferKeyPathConfiguration()
 
     private static let extendedMode = ["restrict_to_standard_functions": false]
+    private static let ignoreIdentity = ["ignore_identity_closures": true]
+    private static let extendedModeAndIgnoreIdentity = [
+        "restrict_to_standard_functions": false,
+        "ignore_identity_closures": true,
+    ]
```

Comment:
This should work, shouldn't it?
```suggestion
    private static let extendedModeAndIgnoreIdentity = extendedMode + ignoreIdentity
```